xbmc/addons/AddonInstaller.cpp

```diff
@@ -479,7 +475,7 @@ int64_t CAddonInstaller::EnumeratePackageFolder(std::map<std::string,CFileItemLi
     std::string pack,dummy;
     AddonVersion::SplitFileName(pack, dummy, items[i]->GetLabel());
     if (result.find(pack) == result.end())
-      result[pack] = new CFileItemList;
+      result[pack] = std::unique_ptr<CFileItemList>(new CFileItemList());
```

Comment:
1. use `make_unique`
2. FYI `()` isn't required. If you want to make it explicit, use `{}`

Reply:
std::make_unique (with the namespace), please